river/rules/amrules.py

```diff
@@ -7,7 +7,7 @@
 from river import base, drift, linear_model, stats, tree
 
 from ..tree.split_criterion import VarianceRatioSplitCriterion
-from ..tree.splitter.base_splitter import Splitter
+from ..tree.splitter.base import Splitter
 from ..tree.splitter.nominal_splitter_reg import NominalSplitterReg
```

Comment:
Meta: at Alan we've decided to not do any relative imports. Here we would do:

```py
from river.tree.split_criterion import VarianceRatioSplitCriterion
```

At some point we can decide if we want to do that too!

Reply:
Nice! We can create a PR to tackle that :)

Is there a specific reason to avoid such imports?

Reply:
They're considered to be messier, see [here](https://realpython.com/absolute-vs-relative-python-imports/). I think the community converged on that in the last few years. One practical thing I find is that I can copy/paste import statements without having to worry where my file is, so I don't have to worry about modifying the import statement 🤷🏼‍♂️

Reply:
Your last argument convinced me haha!

Let's do that soon!